include/onnxruntime/core/graph/graph.h

```diff
@@ -1129,12 +1120,13 @@ class Graph {
   */
   Status InlineFunction(Node& node);
 
-  /** Initialize function body for the given node */
-  void InitFunctionBodyForNode(Node& node);
-
   /** Gets Model local functions from the root/parent graph.*/
   const std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*>& GetModelLocalFunctions() const;
 
+  const std::unordered_map<std::string, FunctionTemplate*>& GetModelLocalFunctionTemplates() const;
```

Comment:
>d::unordered_map<std::string, FunctionTempla [](http://example.com/codeflow?start=10&length=44)

We aim to use InlinedHashMap typedef instead of unordere_map now, to reduce memory allocations. It is now part of the coding standards.

Reply:
sure. will apply to all the changes.